src/controls/dimension-slider.html

```diff
@@ -1,5 +1,5 @@
 <template>
-    <template if.bind="dim === 'z'">
+    <template if.bind="dim === 'z' && image_config.image_info.dimensions.max_z > 1">
```

Comment:
@will-moore: please no nested template tags, they are in 9/10 cases not needed. They are sort of crutches if you don't want the expression/condition to be on the element itself. I use it in the slider because I don't want to introduce another level of element there and in the rois table because of the row-layout and nestedness of the html.

for the purpose of conditionally rendering the slider it's good enough to put that if expression that you have in the template tag on the dimension-slider element in the ol3-viewer.html (one level up so to speak). 

If you want revert it and I'm going to prepare a commit for either cherry-picking or you can use the code.